src/nl/hannahsten/texifyidea/inspections/latex/typesetting/spacing/LatexNonBreakingSpaceInspection.kt

```diff
@@ -54,7 +54,7 @@ open class LatexNonBreakingSpaceInspection : TexifyInspectionBase() {
     /**
      * Commands redefined by cleveref, such that no non-breaking space is needed anymore.
      */
-    private val cleverefRedefinitions = setOf(THREF, VREF, VREFRANGE, FULLREF).map { it.commandWithSlash }
+    private val cleverefRedefinitions = setOf(THREF, VREF, VREF_CAPITAL, VREFRANGE, FULLREF).map { it.commandWithSlash }
```

Comment:
Thanks! I think all of the commands in this list have a capitalized and starred version? But I'm not quite sure if all of them do not require a non-breaking space (if used with cleveref).

If they do not, using `setOf(...).map { it.command }.flatMap { c -> listOf(c, c + "*", c.replaceFirstChar { it.uppercase() }) }.map { "\\" + it } ` also does the job I guess

Reply:
I looked at the list in cleveref's documentation, and `\thref` doesn't seem to have a starred or capital version (but the others do, e.g. `\vref`, `\Vref`, `\vref*` and `\Vref*`), but the support for all those isn't yet available in the big list of commands.

Reply:
Thanks for checking, then `thref` can be left out of the `.map { ... }` part. I know but they don't need to be in the big list of commands, as they should pop up in the autocomplete automatically (see screenshot). The only reason this line is using commands from the big list is so that it's easy to search for code usages of a particular command, that's all.

![image](https://github.com/Hannah-Sten/TeXiFy-IDEA/assets/15669080/1919408a-9788-4464-a6b4-a2b4eee34757)